src/pages/Login.vue

```diff
@@ -275,12 +273,14 @@ export default {
         iconClass: "animate-spin h-4 object-scale-down text-white",
       };
     },
+    /**
+     * whether resendOTPTimer is enabled for resend OTP

```

Comment:
this is the final use case of this right?
```suggestion
     * whether the button for resending OTP is enabled
```

Reply:
yes